redisjson/src/main/java/site/ycsb/db/RedisJSONClient.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright (c) 2020 Redis Labs. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * Redis JSON client binding for YCSB.
+ *
+ * All YCSB records are mapped to a Redis JSON structure; table is ignored and
+ * key is used as the primary index. All fields are located at the root path of the JSON
+ * DOM and values are mapped thereafter as simple scalars due to JReJSON's inability to
+ * instantiate proper POJOs from stored JSON objects.
+ *
+ * For scanning operations, all keys are saved a sorted set with an equal score allowing for lexigraphical retrieval.
+ */
+
+package site.ycsb.db;
```

Comment:
please put this in a binding specific package, e.g. `site.ycsb.db.redisjson`

Reply:
Done. Do you need a new PR?

Reply:
just push commits to the branch for this PR, please.

Reply:
Done